proxysql-scheduler/local/build-binary-proxysql-scheduler

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+
+# ------------------------------------------------------------------------------
+# Declare all input variables
+# ------------------------------------------------------------------------------
+
+
+# ------------------------------------------------------------------------------
+# set working dir
+# ------------------------------------------------------------------------------
+#mkdir -p ${1:-./build}
+WORKDIR=$(cd ${1:-./build}; pwd -P)
+INSTALL_DIR=${WORKDIR}/DESTDIR
+SOURCEDIR=$(cd ${2:-$(dirname $0)/../sources/proxysql-scheduler}; pwd -P)
+
+pushd /tmp
+    #sudo wget -q https://golang.org/dl/go1.15.8.linux-amd64.tar.gz
+    #sudo tar -C /usr/local -xzf go1.15.8.linux-amd64.tar.gz
+    #export PATH=$PATH:/usr/local/go/bin
+    #sudo /usr/local/go/bin/go version
+    #sudo rm go1.15.8.linux-amd64.tar.gz
+    sudo apt update
+    sudo apt install -y golang
```

Comment:
This block of code should be removed, as we don't install build/test dependencies on test-runs,
But this is not critical at this moment, we can put these deps to docker image a little later

Reply:
I agree that it would be best to have a docker image already containing all dependencies. But let's wait for this project to stabilize a little, then go back to this.